src/main/java/domain/models/Account.java

```diff
@@ -10,12 +10,14 @@ public class Account {
     private String password;
 
     public Account(char[] login, char[] password) {
-        try {
-            this.login = new String(Encryption.encryptionSHA256(login));
-            this.password = new String(Encryption.encryptionSHA256(password));
-        } catch (NoSuchAlgorithmException e) {
-            e.printStackTrace();
-        }
+//        try {
+//            this.login = new String(Encryption.encryptionSHA256(login));
+//            this.password = new String(Encryption.encryptionSHA256(password));
+//        } catch (NoSuchAlgorithmException e) {
+//            e.printStackTrace();
+//        }
```

Comment:
Good. But I this uncomment after merge because encryption is work when user register in system.